ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Byron.hs

```diff
@@ -165,6 +167,22 @@ instance (ByronGiven, Typeable cfg) => Measured BlockMeasure (ByronBlock cfg) wh
 
 instance StandardHash (ByronBlock cfg)
 
+{-------------------------------------------------------------------------------
+  Secrets
+-------------------------------------------------------------------------------}
+
+-- This is specifically including Pbft in the name, because Byron Classic
+-- also required a VSS keypair, which Pbft does not.
+--
+-- Alternatively, this type could have been placed in the PBFT protocol,
+-- under a different name, but it would have been even more unappropriately
+-- over-specific there.
+data StaticKeyMaterialByronPbft = StaticKeyMaterialByronPbft {
+      skmbpSignKey     :: Crypto.SigningKey
+      -- | Concrete, because `cardano-crypto` does not provide an abstract delegation certificate.
```

Comment:
API docs should first say what it is, what it is for, and lastly why it is the way it is. This is just a cryptic comment on why something is the way it is (without saying what that alternative is).

Reply:
Done.